cmd/presets/parser.go

```diff
@@ -29,6 +30,21 @@ func (p *DefaultParser) Exists(preset string) (exists bool) {
 	return
 }
 
+// ErrCreateCommandtNotFoundOrEmpty error throwed when did not find the preset create command or it's empty
+var ErrCreateCommandtNotFoundOrEmpty = errors.New("create command not found or empty")
+
+// GetCreateCommand gets the command to create a new project
+func (p *DefaultParser) GetCreateCommand(preset string) (cmd string, err error) {
```

Comment:
just a thought for a later iteration - there might be some preset which requires more than a single command to "create" a new project... we probably could make this abstract, like we do for kool.yml `scripts:  task: single | multiline` do the same for the `preset_create`.

Reply:
And then here for example we could already return a Builder or something, to already encapsulate the runnable create commands.